### DISCUSSION TITLE
Webrtc component

### DISCUSSION THREAD

**Discussion by swatitupat123** [vladmandic/human#384] [Q&A]
Is webrtc being used in this entire implementation ?
If Yes, then where it is being used exactly can you tell the file name ?

**Comment by vladmandic** [comment]
library itself doesn't care about webrtc as once webrtc is initialized, its just a simpler `video` tag like any other  
and yes, `human` works with such initialized video tag as input source.

so question is how to initialize `webrtc` - main demo does have that capability and implementation is in `demo/helpers/webrtc.js`
